src/tests/createHeadlessForm.test.js

```diff
@@ -1117,6 +1117,29 @@ describe('createHeadlessForm', () => {
       });
     });
 
+    it('support format date with minDate and maxDate without the format property', () => {
+      const schemaFormatDate = {
+        properties: {
+          birthdate: {
+            title: 'Birthdate',
+            type: 'string',
+            'x-jsf-presentation': {
+              inputType: 'date',
```

Comment:
@gabrielseco, if I understood correctly, this solution is a workaround to an invalid JSON Schema that does not have `format: date`, right? 
If so, that means that our JSON Schemas are not correct and the data received is not being validated correctly in the BE (because the format is missing, so any string is valid).

In other words, this PR isn't the right solution. Instead, we should fix the incorrect JSON Schemas we have internally.

Does this make sense to you?

Reply:
Yes, I thought that maybe the format property was optional before my change and was not present in all json-schemas.
> this solution is a workaround to an invalid JSON Schema that does not have format: date, right?

Let's close this MR, then